docs/docs/behaviors/sticky-key.md

```diff
@@ -7,7 +7,11 @@ sidebar_label: Sticky Key
 
 A sticky key stays pressed until another key is pressed. It is often used for 'sticky shift'. By using a sticky shift, you don't have to hold the shift key to write a capital.
 
-By default, sticky keys stay pressed for a second if you don't press any other key. You can configure this with the `release-after-ms` setting (see below).
```

Comment:
Can we move the description of the properties to the configuration section below (before the two example snippets in it)? We can then prepend each paragraph with an header with the property name like 

```
### `release-after-ms`
```

This would make it more consistent with the other behavior doc pages like [`hold-tap`](https://zmk.dev/docs/behaviors/hold-tap/) and [`tap-dance`](https://zmk.dev/docs/behaviors/tap-dance).

Reply:
I'll do this. I wasn't sure how to format the document, so those examples help a lot.